src/components/Footer/Footer.js

```diff
@@ -0,0 +1,21 @@
+import logo from "../../images/websiteLogo.png";
+
+function Footer() {
+  return (
+    <footer className="footer footer-center p-10 bg-transparent text-white md:mt-12">
+      <aside>
+        <img src={logo} className="h-[70px] w-[70px]" />
```

Comment:
Every tag `img` should have an informative attribute `alt` (the **name** of the image is usually there) to show the `alternative` text if the image is not uploaded and it’s needed for **screen readers**